### DISCUSSION TITLE
How to save new HasOne and HasMany relations using Model Binding?

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [livewire/livewire#3765] [1. Help]
So I have `Form` in my Livewire Component and the fields on the Form are bound to a Model. The form has fields and a grid that are part of a `HasOne` and HasMany relation. I have set the `wire:model` to these form fields, but I am not sure how when saving the primary model automatically creates new instances of the HasOne and HasMany relations. I can't seem to find any good examples on how to do this on the Livewire docs. Could someone advice on how to do this?

**Comment by joshhanley** [comment]
@abishekrsrikaanth can you provide some concrete examples of what you are trying to do and any code you have tried? It's hard to demonstrate without any model names or anything. 🙂

Thanks!
